src/linker/Linker/Driver.cs

```diff
@@ -700,9 +772,7 @@ protected int SetupContext (ILogger customLogger = null)
 			//
 			// Pipeline setup with all steps enabled
 			//
-			// ResolveFromAssemblyStep [optional, possibly many]
-			// ResolveFromXmlStep [optional, possibly many]
-			// [mono only] ResolveFromXApiStep [optional, possibly many]
```

Comment:
nit: instead of removing these, could you indicate that they're mono only?

Reply:
tweaked the text differently